### PR TITLE
Make database connections configurable by environment variables

### PR DESCRIPTION
**Pull request by KitaitiMakoto**
so that we can tune performance.

### CODE REVIEW

File: plume-models/src/config.rs [hunk 1]
```diff
@@ -193,6 +195,11 @@ lazy_static! {
             var("ROCKET_PORT").unwrap_or_else(|_| "7878".to_owned())
         )),
         db_name: DB_NAME,
+        db_max_size: var("DB_MAX_SIZE").map_or(None, |s| Some(
+            s.parse::<u32>()
+                .expect("Couldn't parse DB_MAX_SIZE into u32")
+        )),
+        db_min_idle: var("DB_MIN_IDLE").map_or(None, |s| s.parse::<u32>().ok()),
```
Comment (elegaanz) [inline plume-models/src/config.rs:202]: Is there a reason for panicking in one case and silently failing in the other?

Comment (KitaitiMakoto) [reply plume-models/src/config.rs:202]: Good catch! There's no reason why. I will rewrite this and push it.

Comment (KitaitiMakoto) [reply plume-models/src/config.rs]: Done.

**Comment by KitaitiMakoto** [comment]
Thank you for the merge.

**Pull request merged**